### PR TITLE
New Keyboard: TGR Jane V2 CE

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

New Keyboard, the TGR Jane V2 CE which is essentially the same as the original TGR Jane V2

I moved around directories and put the original v2 in its own. 

Special thanks to frostfiree for letting me poke at his busted PCB. Special thanks to Xerpocalypse for testing the firmware. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by zvecr** [comment]
Moving and existing keyboard is still considered a "breaking change", so i have re-targeted the PR to develop.

File: keyboards/tgr/jane/v2ce/keymaps/via/config.h [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#define DYNAMIC_KEYMAP_LAYER_COUNT 3
```
Comment (drashna) [inline keyboards/tgr/jane/v2ce/keymaps/via/config.h:17]:
```suggestion
#pragma once

#define DYNAMIC_KEYMAP_LAYER_COUNT 3
```

File: keyboards/tgr/jane/v2ce/v2ce.c [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    setPinOutput(D6);
+}
+
+bool led_update_kb(led_t led_state) {
+    if (led_update_user(led_state)) {
+        writePin(D1, led_state.caps_lock);
+        writePin(D6, led_state.scroll_lock);
+    }
+    return true;
+}
```
Comment (drashna) [inline keyboards/tgr/jane/v2ce/v2ce.c:35]:
```suggestion
```

File: keyboards/tgr/jane/v2ce/config.h [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+//                        0   1   2   3   4   5   6   7   8   9   A   B   C   D   E
+#define MATRIX_ROW_PINS { B0, B1, B2, B3, B4, B5, B6, B7 }
+#define MATRIX_COL_PINS { A0, A1, A2, A3, A4, A5, A6, A7, C7, C6, C5, C4, C3, C2, D7 }
+#define UNUSED_PINS
+
+#define DIODE_DIRECTION COL2ROW
+#define DEBOUNCE 5
+
+#define BACKLIGHT_PIN D4
+#define BACKLIGHT_LEVELS 3
```
Comment (drashna) [inline keyboards/tgr/jane/v2ce/config.h:39]:
```suggestion
#define BACKLIGHT_LEVELS 3

#define LED_CAPS_LOCK_PIN D1
#define LED_NUM_LOCK_PIN D6
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
Has some merge conflicts

**Review by drashna — APPROVED** [review]

File: keyboards/tgr/jane/v2ce/keymaps/default/config.h [hunk 4]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2018 amnesia0287
```
Comment (fauxpark) [inline keyboards/tgr/jane/v2ce/keymaps/default/config.h:1]: This file can be deleted.

File: keyboards/tgr/jane/v2ce/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[0] = LAYOUT_tkl_ansi_f13(\
+  KC_ESC,  KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9, KC_F10, KC_F11, KC_F12,   KC_F13, KC_PSCR,KC_SLCK,KC_PAUS, \
+  KC_GRV,   KC_1,   KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,   KC_0,KC_MINS, KC_EQL,KC_BSPC,   KC_INS ,KC_HOME,KC_PGUP, \
+  KC_TAB,   KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,KC_LBRC,KC_RBRC,KC_BSLS,   KC_DEL ,KC_END ,KC_PGDN, \
+  KC_CAPS,   KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,KC_SCLN,KC_QUOT,         KC_ENT,                          \
+  KC_LSFT,    KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,KC_COMM, KC_DOT,KC_SLSH,        KC_RSFT,            KC_UP,        \
+  KC_LCTL,KC_LGUI,KC_LALT,                 KC_SPC,                                KC_RALT,KC_RGUI, KC_APP,KC_RCTL,  KC_LEFT,KC_DOWN,KC_RGHT) \
```
Comment (fauxpark) [inline keyboards/tgr/jane/v2ce/keymaps/default/keymap.c:26]:
```suggestion
[0] = LAYOUT_tkl_ansi_f13(
  KC_ESC,  KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9, KC_F10, KC_F11, KC_F12,   KC_F13, KC_PSCR,KC_SLCK,KC_PAUS,
  KC_GRV,   KC_1,   KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,   KC_0,KC_MINS, KC_EQL,KC_BSPC,   KC_INS ,KC_HOME,KC_PGUP,
  KC_TAB,   KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,KC_LBRC,KC_RBRC,KC_BSLS,   KC_DEL ,KC_END ,KC_PGDN,
  KC_CAPS,   KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,KC_SCLN,KC_QUOT,         KC_ENT,
  KC_LSFT,    KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,KC_COMM, KC_DOT,KC_SLSH,        KC_RSFT,            KC_UP,
  KC_LCTL,KC_LGUI,KC_LALT,                 KC_SPC,                                KC_RALT,KC_RGUI, KC_APP,KC_RCTL,  KC_LEFT,KC_DOWN,KC_RGHT)
```

File: keyboards/tgr/jane/v2ce/rules.mk [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+MCU = atmega32a
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/tgr/jane/v2ce/rules.mk:11]:
```suggestion
```

File: keyboards/tgr/jane/v2ce/rules.mk [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+# build options
+BOOTMAGIC_ENABLE = no
+MOUSEKEY_ENABLE = yes
+EXTRAKEY_ENABLE = yes
+CONSOLE_ENABLE = no
+COMMAND_ENABLE = yes
+BACKLIGHT_ENABLE = yes
+RGBLIGHT_ENABLE = no
+WS2812_DRIVER = i2c
```
Comment (fauxpark) [inline keyboards/tgr/jane/v2ce/rules.mk:22]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
WS2812_DRIVER = i2c
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**